Tests/ImagineEngineTests/Mocks/TextureErrorHandlerMock.swift

```diff
@@ -0,0 +1,25 @@
+//
```

Comment:
Use the same start-of-file comment format as in the other files here

Reply:
Fixed.  👍